### ISSUE TITLE
Display apparent message for 'too many devices'

### ISSUE THREAD

**Issue by hitzhangjie** [getlantern/lantern#5187]
Please display apparent message for 'too many devices' error, what's the maximum number of devices? Tell us! Why I can't add another one? I used one pc at a time, add more devices is just an convenient manner.
I am a little confused by lantern's operation, I don't have confidence to continue with this solution. Disappointed.
Actually, I didn't add too many devices, I just add several devices with more than one Operating Systems, one device is installed 3 Operating Systems, is it too many devices?

**Comment by hitzhangjie** [comment]
If I have login lantern account in different pc and OS, how can I diff the different devices, the displayed devices list only says 'windows 7, linux,...', I have to remove most of them and validate again!

**Comment by myleshorton** [comment]
You can link 3 devices, and yes Lantern will currently consider virtual machines as additional devices.

I'm curious about your use case. Why do you want to link you virtual machines? You can also unlink devices at any time, but it sounds like the names are conflicting for you, is that right @hitzhangjie?

**Comment by hitzhangjie** [comment]
Yes, I cannot differentiate them, because their names are very similar or
identical. I am not link it in virtual machines, I installed some parallel
operating systems.

*Have a nice day !*

*Zhang Jie*
*Shenzhen, China*

2016-08-22 13:50 GMT+08:00 Myles Horton <notifications@github.com>:

**Comment by sunshine19090** [comment]
#5056
